fastlane_core/lib/fastlane_core/helper.rb

```diff
@@ -12,11 +12,34 @@ def self.log
 
     # Runs a given command using backticks (`)
     # and prints them out using the UI.command method
-    def self.backticks(command, print: true)
-      UI.command(command) if print
-      result = `#{command}`
-      UI.command_output(result) if print
-      return result
+    def self.backticks(command, print: true, retriable_options: {})
+      require 'retriable'
```

Comment:
I'd love to keep the old code, and only use the new one if `retriable_options` is passed. Something like a big `if else` in the beginning of the method